.eslintrc.js

```diff
@@ -0,0 +1,56 @@
+module.exports = {
```

Comment:
I've been thinking about this too. It would require making the eslint file its own artifact with a versioning strategy, and so the process of changing rules would be a bit more complicated, but it would allow us to keep everything in sync, which is probably what we want.